docs/conf.py

```diff
@@ -335,7 +337,9 @@
         "reference_url": {
             "astropy": None,
             "matplotlib": "https://matplotlib.org/stable/",
-            "numpy": "https://numpy.org/doc/stable/",
+            # The numpy search js isn't loadable at the moment (2022-12-07)
+            # It seems to be valid js but it's not valid json so sphinx wont load it.
+            # "numpy": "https://numpy.org/doc/stable/",
```

Comment:
Other than the sphinx version bump I honestly have no idea.

Reply:
This is not a good sign... has this been reported upstream? We should at least open up a follow-up issue, though I guess not critical since this only affects intersphinx in sphinx gallery.

Reply:
Yes, I think I should try offline to run without it so I have something to report to numpy.

Reply:
I get this warning locally:
```
WARNING: The following HTTPError has occurred fetching https://numpy.org/doc/stable//_static/documentation_options.js: 503 (Service Temporarily Unavailable)
```
but https://numpy.org/doc/stable//_static/documentation_options.js works in my browser ...

Reply:
Hmm, I think we web site is also flaky -- I had to reload at times. But sphinx gave a different warning:
```
Extension error (sphinx_gallery.docs_resolv):
Handler <function embed_code_links at 0x7f8ebd5f6a60> for event 'build-finished' threw an exception (exception: Expecting property name enclosed in double quotes: line 1 column 2 (char 1))
```
I think I saw that before.

Note that if I temporarily change to `"numpy": "https://numpy.org/devdocs/"`, it does seem to work.

I'll push that up.

Reply:
This error is what I was getting fwiw.